icu4c/source/i18n/smpdtfmt.cpp

```diff
@@ -2126,11 +2080,6 @@ void SimpleDateFormat::adoptNumberFormat(NumberFormat *formatToAdopt) {
         freeSharedNumberFormatters(fSharedNumberFormatters);
         fSharedNumberFormatters = NULL;
     }
-
-    // Also re-compute the fast formatters.
```

Comment:
Question: should we re-initialize fSimpleNumberFormatter here, like we were doing before? Is there somewhere else they are being re-initialized?